napari/layers/points/_points_utils.py

```diff
@@ -1,10 +1,51 @@
-from typing import Optional, Tuple
+from typing import List, Optional, Tuple
 
 import numpy as np
 
+from ...utils.geometry import project_points_onto_plane
 from ...utils.translations import trans
 
 
+def _create_box_from_corners_3d(
+    box_corners: np.ndarray, box_normal: np.ndarray, up_vector: np.ndarray
+) -> np.ndarray:
+    """Get the corners for a box in 3D from the corners, the normal direction,
+    and the up direction.
```

Comment:
I'm finding it a little hard to grasp exactly what this method does, is it finding the corner positions for a 2d cuboid in 3D? These geometry things are always weird to describe

Reply:
I've updated the docstring as below. Does this seem more clear?

```
    """Get the corners for a box in 3D from two opposing corners, the normal direction,
    and the up direction.

    The resulting box will include the two corners passed in as
    box_corners, lie in a plane with normal box_normal, and have one of its
    axes aligned with the up_vector.

    Parameters
    ----------
    """"

Reply:
Much clearer, thanks!